### ISSUE TITLE
[Feature Request]: Model Layers 

### ISSUE THREAD

**Issue by MeAlam1** [NerdyPuzzle/MCreator-Nerdys_Geckolib_Plugin#90]
I would appreciate it if you would consider adding Model Layers to the Entity's. 
As you might know, Model Layers were added in 2023.4 but we are still missing them in Geckolib.

**Comment by HowToOwnADragon** [comment]
Yes, please. This would make my Variant Loader a lot more Optimized 😄
